tests/test_filtering.py

```diff
@@ -200,7 +302,62 @@ def test_calculate_likelihood_psiphi_zero_or_negative_phi(self):
         # test zero
         lh = calculate_likelihood_psi_phi([1.0], [0.0])
         self.assertEqual(lh, 0.0)
-
+        
+    def test_apply_clipped_average_single_thread(self):      
+        kb_post_process = PostProcess(self.config)
+        
+        res = kb_post_process.apply_clipped_average(self.old_results, None, {}, 0.5)
+        
+        self.assertEqual(len(res), 3)
+        for r in res:
+            self.assertEqual(r[1], [-1])
```

Comment:
I'm not sure that I'm following this assert. I'm probably missing something. What is r[1]? From the code it looks like it is the indices that pass the filtering. But, if that's the case, how does it equal -1?

Reply:
[-1] is the list of passing indices for a given psi/phi curve when it's rejected, all of the results for these particular test curves are filtered out because that's easier to test but I think you're right that there should be a mix of passes and rejections for each test.

Reply:
If we could include a curve where not all results are filtered (it could have all results pass), that would give us better coverage on the tests.